cmd/kops-controller/pkg/controller/controller.go

```diff
@@ -14,34 +13,21 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package controller
```

Comment:
I was going to say, I thought this code looked familiar, great to see it being reused!